test/vm.install

```diff
@@ -46,6 +46,14 @@ fi
 
 systemctl enable cockpit.socket
 
+# HACK: systemd kills the services after 90s
+# See https://github.com/containers/podman/issues/8751
+mkdir -p /etc/systemd/system /etc/systemd/user
+cp /usr/lib/systemd/system/podman.service /etc/systemd/system/podman.service
+cp /usr/lib/systemd/user/podman.service /etc/systemd/user/podman.service
+sed -i "s/Type=notify/Type=exec/" /etc/systemd/system/podman.service
+sed -i "s/Type=notify/Type=exec/" /etc/systemd/user/podman.service
```

Comment:
Not important at all, just in general for next time: Have sed output to the new file instead of copy/in-place edit.

Reply:
oh duh, I was not thinking straight when I wrote this. If I have to repush I'll do that this way.